src/components/KeyboardLayoutDialog.js

```diff
@@ -0,0 +1,100 @@
+import { withTheme } from "@material-ui/core/styles";
+
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import Select from "@material-ui/core/Select";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import useStorage from "../hooks/useStorage";
+import { MenuItem } from "@material-ui/core";
```

Comment:
Oops, a tiny nit - this should be a direct import for [tree shaking](https://material-ui.com/guides/minimizing-bundle-size/) reasons!